src/satchip/chip_sentinel1rtc.py

```diff
@@ -78,23 +82,24 @@ def _get_pct_intersect(product: asf.S1Product, roi: shapely.geometry.Polygon) ->
     return intersection
 
 
-def _get_rtcs_for(slcs_for_chips: dict[str, list[asf.S1Product]], scratch_dir: Path) -> dict[str, list[Path]]:
+def _get_rtcs_for(
+    slcs_for_chips: dict[str, list[asf.S1Product]], image_dir: Path
+) -> dict[str, list[utils.RtcImageSet]]:
     flat_slcs = sum(slcs_for_chips.values(), [])
     slc_names = set(granule.properties['sceneName'] for granule in flat_slcs)
 
     finished_rtc_jobs = _process_rtcs(slc_names)
 
-    paths_for_slc_name: dict[str, Path] = {}
+    paths_for_slc_name: dict[str, utils.RtcImageSet] = {}
     for job in finished_rtc_jobs:
-        rtc_path = _download_hyp3_rtc(job, scratch_dir)
-        slc_name = job.job_parameters['granules'][0]
-
-        paths_for_slc_name[slc_name] = rtc_path
+        rtc_image_set = _download_hyp3_rtc(job, image_dir)
+        slc_name = job.job_parameters['granules'][0]  # type: ignore
+        paths_for_slc_name[slc_name] = rtc_image_set
 
-    rtc_paths_for_chips: dict[str, list[Path]] = {}
+    rtc_paths_for_chips: dict[str, list[utils.RtcImageSet]] = {}
     for chip_name, chip_slcs in slcs_for_chips.items():
-        rtc_paths = [paths_for_slc_name[name.properties['sceneName']] for name in chip_slcs]
-        rtc_paths_for_chips[chip_name] = rtc_paths
+        image_sets = [paths_for_slc_name[name.properties['sceneName']] for name in chip_slcs]
+        rtc_paths_for_chips[chip_name] = image_sets
```

Comment:
What about renaming these to say `image_set` instead of `paths` in the variable names (like `paths_for_slc_name` -> `image_set_for_slc_name`)